tests/resources/test_resources_search.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023-2024 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Tests record search."""
+
+from invenio_rdm_records.records.api import RDMRecord
+
+
+def test_advance_record_search(
+    client,
+    headers,
+    record_community,
+    minimal_record,
+    uploader,
+):
+    """Test search record's communities."""
```

Comment:
Could you please also add a test with punctuation? For example, searching by just `title` and getting no results, then trying to search by `title.original`?

Reply:
Looks like done :)